common/djangoapps/student/models.py

```diff
@@ -284,6 +279,50 @@ def set_login_session(self, session_id=None):
         self.set_meta(meta)
         self.save()
 
+    def requires_parental_consent(self, date=None, age_limit=None, default_requires_consent=True):
+        """Returns true if this user requires parental consent.
+
+        Args:
+            date (Date): The date for which consent needs to be tested (defaults to now).
+            age_limit (int): The age limit at which parental consent is no longer required.
```

Comment:
@andy-armstrong Oddly enough, you missed this usage of "Bool" when changing to "bool".